src/components/CurrentWeek.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import DaySchedule from "./DaySchedule";
+import RestOfWeek from "./RestOfWeek";
+
+const CurrentWeek = ({ sortedSchedule }) => {
+  var dateSet = new Set();
+  sortedSchedule.currentWeek.map((activity) => {
```

Comment:
For what you're doing here, `.forEach` is preferred over `.map`.

Read the differences here:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach